core/src/main/scala/ai/h2o/sparkling/backend/external/RestCommunication.scala

```diff
@@ -209,13 +190,17 @@ trait RestCommunication extends Logging {
     }
   }
 
-  protected def readURLContent(endpoint: URI, requestType: String, suffix: String, conf: H2OConf, params: Map[String, Any] = Map.empty): InputStream = {
-    val suffixWithParams = if (params.nonEmpty && (requestType == "GET")) s"$suffix?${decodeParams(params)}" else suffix
+  protected def readURLContent(endpoint: URI,
+                               requestType: String,
+                               suffix: String,
+                               conf: H2OConf, params: Map[String, Any] = Map.empty,
```

Comment:
nit: `params: Map[String, Any] = Map.empty` to separate line

Reply:
Good catch!